tests/unit-tests/UnitTests/Services/BookingServiceTests.cs

```diff
@@ -81,25 +82,42 @@ public BookingServiceTests()
         [Fact]
         public async Task Get_ShouldRetrieve_BookingInformation_FromFormData_WhenStoredInCache()
```

Comment:
can we have a some new tests to verify MapBookingId is called from MapAppointmentId from the Get and ProcessMonthRequest methods

Reply:
Just added some tests to verify MapBookingId is called within Get and ProcessMonthRequest